tests/e2e/helpers/dumpinfo.go

```diff
@@ -95,10 +94,7 @@ func CreateExportDir(ctx context.Context, t *testing.T) (context.Context, error)
 		return ctx, nil
 	}
 
-	dir, err = ioutil.TempDir("", fmt.Sprintf("tetragon.e2e.%s.*", t.Name()))
-	if err != nil {
-		return ctx, err
-	}
+	dir = t.TempDir()
```

Comment:
Our CI jobs rely on this path being predictable to upload dumps when the job fails. So we will need a way to do the same here. And also a way to ensure the file is kept around when the test fails.